sopel/tools/time.py

```diff
@@ -52,7 +45,7 @@ def validate_format(tformat):
         time = datetime.datetime.utcnow()
         time.strftime(tformat)
     except Exception:  # TODO: Be specific
```

Comment:
While we are here, I tried several format and the expected errors are ValueError (invalid format) and TypeError (not a string):

```suggestion
     except (ValueError, TypeError):
```